### PR TITLE
Set WP_DISABLE_FATAL_ERROR_HANDLER from .env

### PR DESCRIPTION
**Pull request by fiskhandlarn**
And set to true by default

### CODE REVIEW

**Comment by vinkla** [comment]
Great addition! Is the constant set to `true` by default? If not, what is the use-case to prevent these kinds of emails out of the box?

**Comment by fiskhandlarn** [comment]
I'm fairly sure it's set to `false` by default (since I kept getting those emails).
I'm not sure what you mean by "use-case", setting this constant to `true` prevents these kinds of emails out of the box. :)

**Comment by fiskhandlarn** [comment]
I advise to use `true` as default though, since most developers already display errors, right?

**Comment by vinkla** [comment]
If this is set to `false` by default I would argue we should set it to `false` as well. This could be confusing to developers not knowing that the feature is turned off. 

If you're using it in your projects I definitely think we should add it to the framework by default letting developers easily add it to their `.env` file as well.

File: src/Application.php [hunk 1]
```diff
@@ -118,6 +118,9 @@ public function run(): void
         // Prevent file edititing from the dashboard.
         define('DISALLOW_FILE_EDIT', env('DISALLOW_FILE_EDIT', true));
 
+        // Disable "Your Site is Experiencing a Technical Issue" emails
+        define('WP_DISABLE_FATAL_ERROR_HANDLER', env('WP_DISABLE_FATAL_ERROR_HANDLER', true));
```
Comment (vinkla) [inline src/Application.php:122]:
```suggestion
        define('WP_DISABLE_FATAL_ERROR_HANDLER', env('WP_DISABLE_FATAL_ERROR_HANDLER', false));
```

**Comment by fiskhandlarn** [comment]
> If this is set to `false` by default I would argue we should set it to `false` as well. This could be confusing to developers not knowing that the feature is turned off.

image

Okay then :D

**Comment by vinkla** [comment]
Thanks for the pull request! 🙇

**Pull request merged**